core/cas-server-core-util/src/test/java/org/apereo/cas/util/BinaryCipherExecutorTests.java

```diff
@@ -22,7 +22,7 @@
     @Test
     public void checkEncodingDecoding() {
```

Comment:
You also should add a test case to ensure existing keys are and can still be supported. Breaking compatibility at this point probably would not be all too well received.